### PR TITLE
Add jobtask support to API

### PR DESCRIPTION
**Pull request by rclough**
Title says it all, pretty simple PR

### CODE REVIEW

File: dagobah/core/dagobah.py [hunk 1]
```diff
@@ -245,7 +245,8 @@ def add_task_to_job(self, job_or_job_name, task_command, task_name=None,
         job.add_task(task_command, task_name, **kwargs)
         job.commit()
 
-    def add_jobtask_to_job(self, job_or_job_name, target_job, task_name=None):
+    def add_jobtask_to_job(self, job_or_job_name, target_job, task_name=None,
+                           **kwargs):
```
Comment (rclough) [inline dagobah/core/dagobah.py]: just to explain this, the API passes a bunch of unused parameters, and also this matches it's related `add_task_to_job` signature better

Comment (thieman) [inline dagobah/core/dagobah.py]: The only extra param looks like `hostname`, which should probably not actually be an option for JobTask? If so, the `**kwargs` here and the `hostname` bit in `api.py` can both go.

Comment (rclough) [inline dagobah/core/dagobah.py]: Good point, I actually hadnt realized that I was the one who copy pasted that arg in the call. Doh

**Comment by rclough** [comment]
Fixed up that hostname passing issue

File: dagobah/daemon/api.py [hunk 2]
```diff
@@ -481,3 +481,21 @@ def import_job():
...
+@login_required
+@api_call
+def add_jobtask_to_job():
+    args = dict(request.form)
+    if not validate_dict(args,
+                         required=['job_name', 'target_job', 'task_name'],
+                         job_name=str,
+                         target_job=str,
+                         task_name=str,
+                         task_target=str):
```
Comment (thieman) [inline dagobah/daemon/api.py]: `task_target` seems unused

**Comment by thieman** [comment]
One thing otherwise :bear:

File: dagobah/daemon/api.py (hunk 2 above)
Comment (rclough) [inline dagobah/daemon/api.py]: yup, another copy-paste overlook :flushed:

**Comment by rclough** [comment]
done!

**Comment by rclough** [comment]
:+1: ?

**Comment by thieman** [comment]
:bear:

**Comment by rclough** [comment]
I'm unsure the implication of bear emoji?

**Pull request merged**